specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json

```diff
@@ -0,0 +1,479 @@
+        "x-ms-examples": {
+          "Get Operation Result for Library": {
+            "$ref": "./examples/Library_GetOperationResult.json"
+          }
+        },
+        "description": "Get Operation result for Library",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/LibraryResource"
+            }
+          },
+          "202": {
+            "description": "Accepted.",
+            "schema": {
+              "$ref": "#/definitions/OperationResult"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/libraries/{libraryName}": {
+      "delete": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_Delete",
+        "x-ms-examples": {
+          "Delete Library": {
+            "$ref": "./examples/Library_Delete.json"
+          }
+        },
+        "description": "Delete Library",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted."
+          },
+          "409": {
+            "description": "Conflict."
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_Get",
+        "description": "Get Library",
+        "x-ms-examples": {
+          "Get Library": {
+            "$ref": "./examples/Library_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/LibraryResource"
+            }
+          },
+          "304": {
+            "description": "Not modified."
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
```

Comment:
>definitions [](start = 3, length = 11)

Can we re-use some definitions in the ARM Swagger specs (I see you also send a PR to update ARM APIs: [https://github.com/Azure/azure-rest-api-specs/pull/11418](https://github.com/Azure/azure-rest-api-specs/pull/11418))? This is what we do in the existing data plane Swagger specs, for example: [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#L47](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#L47) . In this way, we only maintain one piece of object scheme definition so we can decrease the risk of inconsistency between ARM and data plane APIs